var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -80,172 +88,192 @@ class NetcdfC(AutotoolsPackage):
     #         description='Enable CDM Remote support')
 
     # The patch for 4.7.0 touches configure.ac. See force_autoreconf below.
-    depends_on('autoconf', type='build', when='@4.7.0,main')
-    depends_on('automake', type='build', when='@4.7.0,main')
-    depends_on('libtool', type='build', when='@4.7.0,main')
+    depends_on("autoconf", type="build", when="@4.7.0,main")
```

Comment:
This type of changes (coming from the major cleanup that Todd did in the authoritative spack repo) will make it difficult for us to pull in selective updates until we get the cleanup commit itself merged across.

Reply:
Yeah, I didn't notice it at first, and didn't know that this was in v0.18.1. I'll rebase my PR on top of that.